### DISCUSSION TITLE
How to avoid this?

### DISCUSSION THREAD

**Discussion by maccali** [vercel/next.js#13445] [Help]
I f user does not have cache a page end offline 
he falls into limbo

![Anotação 2020-05-27 113048](https://user-images.githubusercontent.com/55025479/83033690-d06c5100-a00d-11ea-9066-96bdfa821ba0.png)

**Comment by timneutkens** [comment]
It's hard to decipher what you're asking for exactly / what you're doing in your application based on 14 words and a screenshot of a Chrome error page.

**Comment by maccali** [comment]
ok, sorry,

I need a way to respond for user a page if user are offline and doesn`t have this page in cache.

This screenshot ocored in this moment.

I need customize my Workbox "next/pwa" or 
or is there another way?

**Comment by maccali** [comment]
this is a repo https://github.com/maccali/Apod

**Comment by maccali** [comment]
someone?
